src/pages/settings/bank-accounts/create/Create.tsx

```diff
@@ -56,38 +56,36 @@ export function Create() {
 
   const handleCancel = () => {
     if (!isFormBusy) {
-      navigate(route('/settings/bank_accounts'));
+      navigate('/settings/bank_accounts');
     }
   };
 
   const handleSave = async (event: FormEvent<HTMLFormElement>) => {
     if (!isFormBusy) {
       event?.preventDefault();
 
-      const toastId = toast.processing();
+      toast.processing();
       setErrors(undefined);
       setIsFormBusy(true);
 
-      try {
-        await request(
-          'POST',
-          endpoint('/api/v1/bank_integrations'),
-          bankAccount
-        );
-        setIsFormBusy(false);
-        toast.success(t('created_bank_account'), { id: toastId });
-        navigate(route('/settings/bank_accounts'));
-      } catch (cachedError) {
-        const error = cachedError as AxiosError;
-        console.error(error);
-        if (error?.response?.status === 422) {
-          setErrors(error?.response?.data?.errors);
-          toast.dismiss();
-        } else {
-          toast.error();
-        }
-        setIsFormBusy(false);
-      }
+      request('POST', endpoint('/api/v1/bank_integrations'), bankAccount)
+        .then(() => {
+          toast.success('created_bank_account');
+
+          queryClient.invalidateQueries('/api/v1/bank_integrations');
+
+          navigate('/settings/bank_accounts');
+        })
+        .catch((error: AxiosError) => {
+          if (error?.response?.status === 422) {
```

Comment:
I am sorry we have to go thru this again, but `error` is not nullable/undefined property here.

Reply:
> I am sorry we have to go thru this again, but `error` is not nullable/undefined property here.

Sorry I just realized the response is just null, I tried removing the optional chaining from all objects and it didn't work. Removed!